yarn.lock

```diff
@@ -1308,7 +1308,9 @@
     to-fast-properties "^2.0.0"
 
 "@backstage/catalog-model@^0.2.0":
-  version "0.3.0"
+  version "0.2.0"
+  resolved "https://registry.npmjs.org/@backstage/catalog-model/-/catalog-model-0.2.0.tgz#e3fe2a4ddeb6a9b6ec480c80cb2b9c39cb245576"
```

Comment:
This shouldn't have an entry in the lockfile either, fixed by switching the catalog-model dep to 0.3.0